.circleci/config.yml

```
@@ -7,12 +7,23 @@ install_hex_rebar: &install_hex_rebar
       mix local.hex --force
       mix local.rebar --force
 
+install_system_deps: &install_system_deps
+  run:
+    name: Install system dependencies
+    command: |
+      apk add build-base
+
+defaults: &defaults
+  working_directory: ~/repo
+
 jobs:
-  build_elixir_1_13_otp_25:
+  build_elixir_1_15_otp_26:
     docker:
-      - image: hexpm/elixir:1.13.4-erlang-25.0-alpine-3.15.4
-    steps:
+      - image: hexpm/elixir:1.15.2-erlang-26.0.2-alpine-3.18.2
+    <<: *defaults
+     steps:
```

```suggestion
    steps:
```
I don't think it likes this indentation being off

---

fixed